src/main/java/org/folio/service/pieces/flows/DefaultPieceFlowsValidator.java

```diff
@@ -56,6 +58,16 @@ public static boolean isCreateHoldingForPiecePossible(Piece pieceToCreate, Compo
                         && PoLineCommonUtil.isHoldingUpdateRequiredForPhysical(originPoLine));
   }
 
+  public static List<Error> isDisplayOnHoldingsNotConsistent(Piece piece) {
+    if (Boolean.FALSE.equals(piece.getDisplayOnHolding()) && Boolean.TRUE.equals(piece.getDisplayToPublic())) {
+      ErrorCodes error = PIECE_DISPLAY_ON_HOLDINGS_IS_NOT_CONSISTENT;
+      return List.of(new Error()
+        .withCode(error.getCode())
+        .withMessage(error.getDescription()));
```

Comment:
this make code less return List.of(PIECE_DISPLAY_ON_HOLDINGS_IS_NOT_CONSISTENT.toError());

Reply:
we can use ternary operation
```
return piece.getDisplayOnHolding() && piece.getDisplayToPublic() ? List.of(PIECE_DISPLAY_ON_HOLDINGS_IS_NOT_CONSISTENT.toError()): 
Collection.EmptyList
```

Reply:
applied suggesting List.of(PIECE_DISPLAY_ON_HOLDINGS_IS_NOT_CONSISTENT.toError());
Regarding second one - kept structure as in similar methods without ternary operation
Sonar complains regarding piece.getDisplayOnHolding() && piece.getDisplayToPublic(), because of this we have this longer construction.
Thanks for suggestions